stratum/hal/lib/phal/tai/tai_wrapper/BUILD

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright 2018 Google LLC
+# Copyright 2018-present Open Networking Foundation
+#
```

Comment:
Fix copyright

Reply:
Fixed copyright headers in all files (recursively) under the `stratum/hal/lib/phal/tai/` path. Please review.